docs/source/python_api/mlflow.models.evaluation.base.rst

```diff
@@ -1,7 +0,0 @@
-mlflow.models.evaluation.base
-=============================
-
-.. automodule:: mlflow.models.evaluation.base
-    :members:
-    :undoc-members:
-    :show-inheritance:
```

Comment:
Note: delete the mlflow.models.evaluation.base doc page. All user level API has been exported to mlflow.models.evaluation module